qa/Vagrantfile

```diff
@@ -17,6 +17,14 @@ Vagrant.configure(2) do |config|
         sh.path = "sys/#{platform.type}/bootstrap.sh"
         sh.privileged = true
       end
+
+      if platform.bootstrap
```

Comment:
not sure I get this way of adding the new specific platform specific scripts here, I see a few open questions for me:

* bootstrap seems to be a confusing name, specially If I only take a look at the Vagrantfile. Is not clear why this file and code is there for.
* On the other side, we now mix, have some scripts defined in the vagrantfile and some in the json file. My take here would be to have either all in the json file or have them defined only in the vagrantfile, otherwise there is an unnecessary mix in the configuration (ruby (vagrantfile) vs json).

What do you think?